src/pages/patientView/trialMatch/TrialMatchTableUtils.ts

```diff
@@ -69,10 +69,10 @@ export function groupTrialMatchesByAgeNumerical(armGroup: ITrialMatch[]): IClini
             }
         };
         const positiveTrialMatches = _.filter(matchesGroupedByAge[age], (trialMatch:ITrialMatch) => {
-            if (!_.isUndefined(trialMatch.genomicAlteration)) return !trialMatch.genomicAlteration.includes('!')
+            if (!_.isUndefined(trialMatch.genomicAlteration) && trialMatch.genomicAlteration !== '') return !trialMatch.genomicAlteration.includes('!')
```

Comment:
always use brackets after if statements:

```suggestion
            if (!_.isUndefined(trialMatch.genomicAlteration) && trialMatch.genomicAlteration !== '') {
                return !trialMatch.genomicAlteration.includes('!')
            }
```

Reply:
Will update my code. Thanks